internal/test/config.go

```diff
@@ -69,8 +69,12 @@ var testGenesisFmt = `{
 		"abci": {
 			"vote_extensions_enable_height": "0"
 		},
-		"version": {}
-	},
+		"version": {},
+		"feature": {
+			"vote_extensions_enable_height": "0",
+			"pbts_enable_height": "0"
```

Comment:
I wonder whether this should be set to 1 here, meaning PBTS enabled?

Reply:
maybe as a second test case

Reply:
Will address in [#2324](https://github.com/cometbft/cometbft/issues/2324)